lib/mix/tasks/phx.gen.auth/injector.ex

```diff
@@ -162,24 +162,27 @@ defmodule Mix.Tasks.Phx.Gen.Auth.Injector do
   def app_layout_menu_code_to_inject(%Schema{} = schema, padding \\ 4, newline \\ "\n") do
     already_injected_str = "#{schema.route_prefix}/log_in"
 
+    base_tailwind_classes = "text-[0.8125rem] leading-6 text-zinc-900 hover:text-zinc-700"
```

Comment:
I'm new to tailwind, so I'm not sure what the best practice is for reusing classes, as these are the classes applied to the links in the header from the phx.new generator. It might make sense to house these classes in a common place, just not sure what they convention would be within a phoenix app.

Reply:
Based on the comment [here](https://github.com/phoenixframework/phoenix/issues/5176#issuecomment-1387733898), sounds like this can be resolved by using the components in core_components. Gonna play around and verify.

Reply:
Okay, so played around with that, and there are no base `link` components defined in core components. The `.link` component comes from Phoenix.LiveView.Helpers, and I'm hesitant to change global link styles across the entire phoenix stack for this change.